python/flashinfer/sampling.py

```diff
@@ -213,6 +213,74 @@ def top_k_sampling_from_probs(
         probs, uniform_samples, top_k, deterministic
     )
 
+def min_p_sampling_from_probs(
+    probs: torch.Tensor,
+    uniform_samples: torch.Tensor,
+    min_p: torch.Tensor,
+    deterministic: bool = True,
+) -> Tuple[torch.Tensor, torch.Tensor]:
+    r"""Fused GPU kernel for min_p sampling from probabilities,
```

Comment:
Add a reference to the definition of min-P sampling.

Reply:
Done with https://github.com/flashinfer-ai/flashinfer/pull/422/commits/43ffad754c31f1356442bbbb4cb83389d40dd089.